### ISSUE TITLE
Where is castor-maven-plugin-2.5 in Central

### ISSUE THREAD

**Issue by awhitford** [mojohaus/castor-maven-plugin#1]
It looks like 2.5 has been released, but I can't find it in Central...  Did the `groupId` change?

Can you comment on this plugin vs this other one:

```
<dependency>
  <groupId>org.codehaus.castor</groupId>
  <artifactId>castor-maven-plugins</artifactId>
  <version>1.4.0</version>
</dependency>
```

**Comment by rfscholte** [comment]
Have a look at the project summary page: http://www.mojohaus.org/castor-maven-plugin/project-summary.html

**Comment by aner-perez** [comment]
I don't see an answer to this question on the project summary page.  The project summary page lists "2.5" as the version but the highest version number available on Maven Central is "2.1".

Looking at the releases tab here on GitHub it says that 2.5 was created on Oct. 1, 2013.  If this is the case, shouldn't it be available on Maven Central?
